static/js/TopicPage.jsx

```diff
@@ -369,21 +371,56 @@ const getTopicHeaderAdminActionButtons = (topicSlug, refTopicLinks) => {
 
     return actionButtons;
 };
+const AiInfoTooltip = () => {
```

Comment:
we want this component to be accessible in multiple places. can you put it in Misc.jsx?